release-tools/check-crate-version/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "check-crate-version"
+version = "0.1.0"
```

Comment:
Can you update the version of this crate to `0.0.0` and add `publish = false`? If decide to publish this, it most likely won't be from this repo.

Reply:
Fixed